webapp/src/main/java/com/box/l10n/mojito/service/screenshot/ScreenshotService.java

```diff
@@ -383,4 +372,35 @@ String escapeAndWrapValueForContains(String value) {
   public void updateScreenshot(Screenshot screenshot) {
     screenshotRepository.save(screenshot);
   }
+
+  /**
+   * Deletes a screenshot
+   *
+   * @param id screenshotId
+   */
+  @Transactional
+  public void deleteScreenshot(Long id) {
+    List<ThirdPartyScreenshot> thirdPartyScreenshots =
+        thirdPartyScreenshotRepository.findAllByScreenshotId(id);
+    if (thirdPartyScreenshots.size() > 0) {
+      for (ThirdPartyScreenshot thirdPartyScreenshot : thirdPartyScreenshots) {
+        String repository =
+            thirdPartyScreenshot.getScreenshot().getScreenshotTextUnits().stream()
+                .findFirst()
+                .get()
+                .getTmTextUnit()
+                .getAsset()
+                .getRepository()
+                .getName();
+        String projectId =
+            env.getProperty(String.format("l10n.link.%s.thirdParty.projectId", repository));
```

Comment:
can this be accessed in the configuration already available/reuse the LinkConfig class?